src/main/kotlin/no/nav/syfo/service/BehandlingService.kt

```diff
@@ -121,24 +122,12 @@ class BehandlingService(
             return null
     }
 
-    suspend fun skalBehandleJournalpost(
-        hendelsesType: String,
-        journalpostId: String,
-        sykmeldingId: String,
-        loggingMeta: LoggingMeta
+    fun skalBehandleJournalpost(
+        hendelsesType: String
     ): Boolean {
-        if (hendelsesType == "MidlertidigJournalført") {
+        if (hendelsesType == "MidlertidigJournalført" || hendelsesType == "TemaEndret") {
```

Comment:
Vil ikke alt vi mottar være enten midlertidig journalført eller tema endret siden det er de to hendelsene vi lytter på lenger opp..? Altså at denne alltid er true?

Reply:
Godt poeng, dette blir overflødig!